api/client.go

```diff
@@ -14,6 +14,20 @@ type Client struct {
 	RestyClient *resty.Client
 }
 
+// InfraClient represents the client state for the Infrastructure API
+type InfraClient struct {
```

Comment:
I'm not as familiar with this part of the New Relic API, so Infra is an entirely separate API (including a separate endpoint) I take it?

Reply:
That's correct—separate endpoint.